shared/utility/motion/splines/Footstep.cpp

```diff
@@ -85,15 +85,15 @@ namespace utility::motion::splines {
         stepFromSupport(tmpDiff);
     }
 
-    Eigen::Vector3f Footstep::poseAdd(const Eigen::Vector3f& pose, const Eigen::Vector3f& diff) const {
+    Eigen::Vector3f Footstep::poseAdd(const Eigen::Vector3f& pose, const Eigen::Vector3f& diff) {
```

Comment:
Removing const?

Reply:
This was made static. Const doesn't make sense for static method